ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -729,7 +729,16 @@ literals:
 - lost_power_alarm:
   - ACTIVE
   - INACTIVE
-
+- refrigerant_failed_alarm:
+  - ACTIVE
+  - INACTIVE
+- outside_heating_exchange_failed_alarm:
```

Comment:
outside a heat exhanger? what does this mean exactly?

Reply:
It is coil responsible for thermal energy exchange failed alarm. Can it be a heat_exchange_failed_alarm .
Done the commit